example/storage/persisted-prometheus.yaml

```diff
@@ -4,7 +4,10 @@ metadata:
   name: persisted
 spec:
   storage:
-    class: ssd
-    resources:
-      requests:
-        storage: 40Gi
+    volumeClaimTemplate:
+      metadata:
+        volume.beta.kubernetes.io/storage-class: ssd
```

Comment:
this is a field of the volumeClaimTemplate called `storageClassName`

Reply:
Thanks, I didn't realize that was available.